src/routes/albumRoute.ts

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+import { AlbumController } from '../controllers/AlbumController';
+
+const router = Router();
+const albumController = new AlbumController();
+
+router.post('/album', albumController.createAlbum.bind(albumController));
+router.get('/album', albumController.getAlbums.bind(albumController));
+router.get('/album/:id', albumController.getAlbumById.bind(albumController));
+router.get('/album/by-artist/:artistId', albumController.getAlbumsByArtistId.bind(albumController));
```

Comment:
You've missed two more routes here:

- `PUT /album/:id`
- `DELETE /album/:id`